ui/app/src/components/PluginCard/PluginCard.tsx

```diff
@@ -38,6 +38,9 @@ interface PluginCardProps {
   plugin: MarketplacePlugin;
   interval?: number;
   marketplace?: boolean;
+  isInstalled?: boolean;
+  installPermission?: boolean;
```

Comment:
Let's call `usePermission` (use applies better to both usages of the component).